tests/lib/snaps/test-snapd-framebuffer/bin/write

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "$1" | dd bs=4 seek=$((100 * 1024 + 200)) >/dev/fb0
```

Comment:
I think you mean:
`echo "$1" | dd bs=4 seek=$((100*1024 + 200)) of=/dev/fb0`

Reply:
done